geopandas/tests/test_pandas_methods.py

```diff
@@ -474,8 +474,14 @@ def test_value_counts():
     # each object is considered unique
     s = GeoSeries([Point(0, 0), Point(1, 1), Point(0, 0)])
     res = s.value_counts()
+    if compat.PANDAS_GE_20:
```

Comment:
Is this change planned for 2.0 or it there a chance it will come as part of 1.x? (Haven't checked pandas side)

Reply:
Forgot to link, https://github.com/pandas-dev/pandas/pull/49912
It's tagged as 2.0 milestone and the readme in the PR marks it as a breaking change of 2.0

Reply:
Cool, it is good to go then.

Reply:
Yes, it's just for 2.0. Thanks for the fix Matt!